client/components/Parts/Trees/File.tsx

```diff
@@ -114,10 +113,9 @@ const StyledTreeItem = styled(
 }))
 
 function TreeItemIcon(props: { nodeId: string; item: types.IFileTreeItem }) {
-  const Icon = getIcon(props.item.type)
-  let color = 'disabled'
+  let color = 'gray'
   if (props.item.type === 'folder') color = 'primary'
-  if (props.item.name) color = props.item.errors ? 'error' : 'success'
+  if (props.item.name) color = props.item.errors ? 'red' : 'green'
```

Comment:
this is still a WiP, will be replaced with the proper colors from the design once it is finished